net/mux/mux.go

```diff
@@ -364,6 +364,8 @@ type Stream struct {
 	established bool      // has the first frame been sent?
 	err         error
 	readBuf     []byte
+	readBytes   uint64
+	writeBytes  uint64
```

Comment:
I don't think this is the right place to be tracking bandwidth metrics. Easy to wrap the conn, like `statsConn` in `us`. Usually you'll want to track other metrics too anyway, and wrapping allows you to gather stats on any type of connection.